transcom/mymove
# B-21626 Update export to view Assigned column all queues

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21626)

## Summary
Ability for SC, TOO and TIO user to export and show Assigned column values.
- Counseling Queue 
- PPM Closeout Queue
- Task Order Queue
- Payment Request Queue

### How to test

1. Login as a customer and create two moves.
2. Login as a Services Counselor, on the service counselor queue assign one of the payment requests
3. Press the export to csv button
4. Verify the report has the correct first and last name for the assigned selection 
5. Verify the report has no value for the move that does not have an assigned selection 
6. submit the move, 
7. login as TOO
8. Assign one of the payment requests
9. repeat steps 3-5
10. approve the move
11. Login as prime and update the shipment
12. create a payment request for both moves
13. Assign one of the payment requests
14. repeat septs 3-5
15. Login as customer and create a PPM move
16. submit move and add shipment
17. login as Service counselor, on the PPM Closeout queue make a selection for the assigned column on one of the moves
18. repeat steps 3-5

## [INT PR](https://github.com/transcom/mymove/pull/14307)
